src/util.ts

```diff
@@ -0,0 +1,87 @@
+import { Logger } from './logger';
+
+const temp = require('temp');
+const fs = require('fs');
+const logger:Logger = Logger.getInstance();
+
+
+export class TempFileHandler {
+    track: any;
+
+    create(TextEditor) {
```

Comment:
It might make more sense to have this accept just the text, regardless of the source. This would keep the Atom specific part in the calling area.

So instead of calling this in `linter.ts:62` with this:

```ts
this.tempFile = this.tempFileHandler.create(textEditor);
```

You would do:

```ts
this.tempFile = this.tempFileHandler.create(textEditor.getText());
```